### PR TITLE
#7340 - remove all the code related to multi-process status 

### LINKED ISSUE #7340: Remove multi-process status (AddonFeatureCompatibility code)
With Firefox 57 everything is multi-process now, so we can probably remove this visual indicator in devhub:

screenshot 2018-01-18 11 11 10

Or perhaps we want to wait for Thunderbird and Seamonkey to move off.

Also there's an API that could maybe go?

http://addons-server.readthedocs.io/en/latest/topics/api/addons.html#add-on-feature-compatibility

### PR DESCRIPTION
**Pull request by ajupazhamayil**
Removed all the code related to multi-process status (AddonFeatureCompatibility code) except message ids in django.po files and deleted files which are no longer needed. (django.po file update is not required for fixing this issue)

Fixes #7340

### CODE REVIEW

**Comment by wagnerand** [comment]
> Sure, I will keep that in the new request. So should I wait for full review or close this and make a new PR right away?

Please don't create a new PR. Instead, you can just add new commits to your branch. As soon as you push that branch, the changes will be reflected in this pull request.

**Comment by ajupazhamayil** [comment]
Thank you for the information @wagnerand. @EnTeQuAk I have restored the migration file src/olympia/migrations/878-add-feature-compatibility-table.sql

**Comment by ajupazhamayil** [comment]
Is it require to remove messages from django.po file in locale folder? If yes please guide me how to do it.

File: src/olympia/migrations/878-add-feature-compatibility-table.sql [hunk 1]
```diff
@@ -6,3 +6,4 @@ CREATE TABLE `addons_addonfeaturecompatibility` (
 )
 ;
 ALTER TABLE `addons_addonfeaturecompatibility` ADD CONSTRAINT `addon_id_refs_id_7779cd14` FOREIGN KEY (`addon_id`) REFERENCES `addons` (`id`);
+
```
Comment (diox) [inline src/olympia/migrations/878-add-feature-compatibility-table.sql]: Please keep this old migration intact, no need to modify it that's confusing.

**Comment by diox** [comment]
There is one thing left to do that I forgot to mention in the issue. You need to add a new migration removing the `addons_addonfeaturecompatibility` table entirely. It should be done in SQL in a file called `src/olympia/migrations/1066-remove-addonfeaturecompatibility.sql`.

**Comment by ajupazhamayil** [comment]
> There is one thing left to do that I forgot to mention in the issue. You need to add a new migration removing the `addons_addonfeaturecompatibility` table entirely. It should be done in SQL in a file called `src/olympia/migrations/1066-remove-addonfeaturecompatibility.sql`.

Thank you for reviewing my PR @diox.
I have created new migration 7340-remove-addonfeaturecompatibility.sql and copied the old 878-add-feature-compatibility-table.sql into migrations.

File: src/olympia/migrations/878-add-feature-compatibility-table.sql (hunk 1 above)
Comment (ajupazhamayil) [reply src/olympia/migrations/878-add-feature-compatibility-table.sql]: I have copied the old migration and committed.

**Comment by diox** [comment]
Looks good, thanks!

**Comment by ajupazhamayil** [comment]
@diox Thank you for approving my First PR! Thank you all!

**Comment by caitmuenster** [comment]
Congrats, @ajupazhamayil! 🎉 Your contribution has been added to our [recognition wiki](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#December_2018). 

Would you be interested in creating an account on [mozillians.org](https://mozillians.org)? I'd be happy to vouch for you!

**Comment by ajupazhamayil** [comment]
Thank you @caitmuenster, I am very much interested to create an account in 
https://mozillians.org/
Please vouch for me : https://mozillians.org/en-US/u/ajupazhamayil/

**Comment by caitmuenster** [comment]
You are vouched! 🎉 Welcome onboard!

**Comment by ajupazhamayil** [comment]
Thank you!

**Pull request merged**